beekeeper-integration-tests/pom.xml

```diff
@@ -19,11 +19,31 @@
       <groupId>com.expediagroup</groupId>
       <artifactId>beekeeper-path-scheduler-apiary</artifactId>
       <version>${project.version}</version>
+      <exclusions>
```

Comment:
this is weird though and I would think the same problem occurs when you run it unless something the Integration tests pull in that doesn't have this. or do we repackage the jars or something?
Would be nice to have a test for the healthcheck :/

Reply:
Fixed, now excluding near the source in the path scheduler. Also I've added integration tests for the healthcheck.

I had to change `assertMertrics()` as we used to assert that the list of metrics was a certain size, but the web dependency adds a whole bunch of extra metrics for free (~90 or so).